R/zcurve.R

```diff
@@ -39,7 +39,7 @@ zcurve <- function(x, zscore) {
   }
 
   ggplot(x) +
-    stat_density(aes(x = !! rlang::enquo(zscore), color = "Observed"), size = 1,
+    stat_density(aes(x = !! rlang::enquo(zscore), color = "Observed"), linewidth = 1,
                  geom = "line") +
     stat_function(fun     = stats::dnorm,
                   args    = list(mean = 0, sd = 1), size = 1,
```

Comment:
@zkamvar this still uses size, is that the issue?

Reply:
and then it'd make sense, depending on how you run the tests, that you don't get the warning every time.

https://github.com/tidyverse/ggplot2/blob/15cb2fbc1de1c906fe5264d3cef85c2a9338f375/R/utilities.r#L717

> warnings are only issued once every 8 hours to avoid overwhelming the user.

https://lifecycle.r-lib.org/reference/deprecate_soft.html

Reply:
But the thing is, this PR fixes that issue. You can see in the diff `size` changes to `linewidth` 🤔

Reply:
OMG I SEE IT NOW 😨